js/open-close-form-alert.js

```diff
@@ -0,0 +1,75 @@
+import { closeForm } from './open-close-upload-form.js';
+import { onClickWithinAlert } from './util.js';
+
+const successTemplate = document.querySelector('#success').content;
+const successBlock = successTemplate.querySelector('.success');
+const closeSuccesseAlertButton = successBlock.querySelector('.success__button');
+const failTemplate = document.querySelector('#error').content;
+const failBlock = failTemplate.querySelector('.error');
+const closeFailAlertButton = failBlock.querySelector('.error__button');
+
+const initSuccesseAlert = () => {
+  document.body.append(successTemplate);
+  successBlock.classList.add('hidden');
+};
+
+const removeSuccesseAlert = () => {
+  successBlock.classList.add('hidden');
```

Comment:
тут и ниже опечатка в названии, буква пропущена 
```suggestion
const removeSuccessesAlert = () => {
  successBlock.classList.add('hidden');
```